index.d.ts

```diff
@@ -1,17 +1,26 @@
 // Definitions by: Jannik <https://github.com/jannikkeye>
 //                 Leo <https://github.com/leomelzer>
-
-import fastify = require("fastify");
-
+/// <reference types="node" />
+import * as fastify from 'fastify';
+import { Plugin } from "fastify";
 import { Server, IncomingMessage, ServerResponse } from "http";
+import { Http2SecureServer, Http2Server, Http2ServerRequest, Http2ServerResponse } from "http2";
+
```

Comment:
You should include the `https` package too i.e `https.Server`. Add it to the server type on line 9. No additions need to be made in the IncomingMessage/ServerResponse as `https` uses the `http` ones.

Reply:
@Ethan-Arrowood "Ethan-Arrowood requested changes" - it's sems that i can't approve your requested changes, becouse this option is restricted for me :/

Reply:
Make your changes, commit, and then I can resolve my requested changes 😄

Reply:
@Ethan-Arrowood ok, i think i did what you asked, just confused about why i need to do it, if you could made same changes as you alredy changed so much ... Sorry for troble, i don't use git that often.